app/src/main/res/values/colors.xml

```diff
@@ -7,4 +7,5 @@
     <color name="teal_700">#FF018786</color>
     <color name="black">#FF000000</color>
     <color name="white">#FFFFFFFF</color>
+    <color name="marvel_red">#ED1D24</color>
 </resources>
```

Comment:
add an empty line at the EoF to avoid this error

Reply:
Solved